gateway/handlers/infohandler.go

```diff
@@ -11,6 +11,8 @@ import (
 	"io/ioutil"
 	"net/http/httptest"
 
+	"syscall"
```

Comment:
@ivanayov I would like to avoid using a syscall. In the issue and PR I mentioned several times that we should use the same approach as the function store where we hard-code the architecture in the Dockerfile, or determine it as a flag and pass it by LDFLAG.

Hard-coded in Dockerfile:

https://github.com/openfaas/faas/blob/master/gateway/Dockerfile.armhf#L59

Passing via ldflag:

https://github.com/openfaas/faas/blob/master/gateway/Dockerfile.armhf#L31

Please can you pick one of the ways which I have asked for originally? I don't mind which it is.